tests/tileset_reader_test_data/white_buildings_with_lods/tileset.json

```diff
@@ -0,0 +1 @@
+{"asset":{"version":"1.0"},"geometricError":500.0,"root":{"boundingVolume":{"box":[1843097.75,5174345.75,177.39999877929688,29.125,0.0,0.0,0.0,25.25,0.0,0.0,0.0,10.0]},"geometricError":500.0,"refine":"ADD","children":[{"boundingVolume":{"box":[1.25,-1.25,0.529998779296875,29.125,0.0,0.0,0.0,25.25,0.0,0.0,0.0,10.0]},"geometricError":20,"refine":"REPLACE","content":{"uri":"tiles/0.b3dm"},"children":[{"boundingVolume":{"box":[1.25,-1.25,0.529998779296875,29.125,0.0,0.0,0.0,25.25,0.0,0.0,0.0,10.0]},"geometricError":5,"refine":"REPLACE","content":{"uri":"tiles/1.b3dm"},"children":[{"boundingVolume":{"box":[1.25,-1.25,0.529998779296875,29.125,0.0,0.0,0.0,25.25,0.0,0.0,0.0,10.0]},"geometricError":1,"refine":"REPLACE","content":{"uri":"tiles/2.b3dm"},"transform":[1.0,0.0,0.0,0.0,0.0,1.0,0.0,0.0,0.0,0.0,1.0,0.0,0.0,0.0,0.0,1.0]}],"transform":[1.0,0.0,0.0,0.0,0.0,1.0,0.0,0.0,0.0,0.0,1.0,0.0,0.0,0.0,0.0,1.0]}],"transform":[1.0,0.0,0.0,0.0,0.0,1.0,0.0,0.0,0.0,0.0,1.0,0.0,1843096.5,5174347.0,176.87,1.0]}]}}
```

Comment:
That line misses an endline. Is this due to the tileset writer (when you produced that testing data set) or is this a manual edit ? If the missing endline is due to the tileset writer maybe that writer should be "fixed" to place a trailing endline ?

Reply:
The tileset.json is written by the [to_json method](https://github.com/VCityTeam/py3dtiles/blob/Tiler/py3dtiles/threedtiles_notion.py#L102) of py3diles with `json.dumps`. This method doesn't add any endlines, but maybe we can add one "manually" after the dumps.

Reply:
Issue open [here](https://github.com/VCityTeam/py3dtiles/issues/21)